src/System.Runtime/ref/System.Runtime.Manual.cs

```diff
@@ -9,24 +9,24 @@ namespace System
 {
     public partial struct Double
     {
-        public const double MinValue = -1.7976931348623157E+308;
         public const double MaxValue = 1.7976931348623157E+308;
+        public const double MinValue = -1.7976931348623157E+308;
 
         // Note Epsilon should be a double whose hex representation is 0x1
         // on little endian machines.
         public const double Epsilon = 4.9406564584124654E-324;
-        public const double NegativeInfinity = (double)-1.0 / (double)(0.0);
-        public const double PositiveInfinity = (double)1.0 / (double)(0.0);
-        public const double NaN = (double)0.0 / (double)0.0;
+        public const double NaN = 0.0 / 0.0;
+        public const double NegativeInfinity = -1.0 / 0.0;
+        public const double PositiveInfinity = 1.0 / 0.0;
     }
     public partial struct Single
     {
-        public const float MinValue = (float)-3.40282346638528859e+38;
-        public const float Epsilon = (float)1.4e-45;
-        public const float MaxValue = (float)3.40282346638528859e+38;
-        public const float PositiveInfinity = (float)1.0 / (float)0.0;
-        public const float NegativeInfinity = (float)-1.0 / (float)0.0;
-        public const float NaN = (float)0.0 / (float)0.0;
+        public const float Epsilon = 1.4E-45f;
```

Comment:
@tannergooding - for some reason the auto-gen changes the value to `1.401298E-45f`.

Similarly, the other values (MaxValue/MinValue) show up incorrectly as well. I kept the values that already exist but not sure what the intention is here:
```C#
public const float MaxValue = 3.40282347E+38f;
public const float MinValue = -3.40282347E+38f;
```

Reply:
I'm guessing because the improved algorithm identifies the shorter forms as bit-equivalent in the floating point representation, and more desirable as they're shorter.

Reply:
The current ref isn't shorter in all cases, and what GenAPI is proposing isn't either.

**Current (and what's in master/this PR):**
```C#
Epsilon = 1.4E-45f;
MaxValue = 3.40282346638528859E+38f;
```

**What GenAPI was suggesting changing this to:**
```C#
Epsilon = 1.401298E-45f; // this is a longer form
public const float MaxValue = 3.40282347E+38f; // this is a shorter form
```

Reply:
Ah, @tannergooding would have the explain that.